Header header
bool pin_one
bool pin_two
